android/KMAPro/build.sh

```diff
@@ -48,6 +73,45 @@ else
   DAEMON_FLAG=
 fi
 
+# Download default keyboard and dictionary
+if [ $DO_DOWNLOAD = true ]; then
+  echo "Downloading default keyboard & dictionary from downloads.keyman.com"
+
+  SCRIPT_ROOT=`pwd`
+  URL_DOWNLOAD=https://downloads.keyman.com
+  URL_API_KEYBOARD_VERSION=${URL_DOWNLOAD}/api/keyboard/
+  URL_API_MODEL_VERSION=${URL_DOWNLOAD}/api/model/
+
+  # Default Keyboard
+  id="sil_euro_latin"
+  echo "Downloading $id"
+  URL_DOWNLOAD_FILE=`curl -s "$URL_API_KEYBOARD_VERSION/$id" | "$JQ" -r .kmp`
+  curl -f -s "$URL_DOWNLOAD_FILE" -o "$ASSETS_KEYBOARD"
+
+  if [ $? -ne 0 ]; then
+      die "Downloading $ASSETS_KEYBOARD failed"
+  fi
+
+  # Default Model
+  id="nrc.en.mtnt"
+  echo "Downloading $id"
+  URL_DOWNLOAD_FILE=`curl -s "$URL_API_MODEL_VERSION/$id" | "$JQ" -r .kmp`
+  curl -f -s "$URL_DOWNLOAD_FILE" -o "$ASSETS_MODEL"
+
+  if [ $? -ne 0 ]; then
+      die "Downloading $ASSETS_MODEL failed"
+  fi
+fi
```

Comment:
Might be worth making this a separate script under `resources/` - at some point, we may wish to update iOS to do the same.  I'd then suggest making `$ASSETS_KEYBOARD` and `$ASSETS_MODEL` parameters of the resulting function.

Reply:
I think this will only be for the Android and iOS apps.
Nonetheless, I've split it into `resources/build/build-download-resources.sh`